Tests/Functional/Components/TransactionReport/TransactionReportTest.php

```diff
@@ -170,10 +170,10 @@ public function testGetReportResult()
         ];
 
         $expectedTurnoverResult = [
-            'EUR' => 14442.3,
-            'DKK' => 11962.28,
-            'CHF' => 12478.41,
-            'USD' => 15057.28,
+            'EUR' => 14170.61,
```

Comment:
Just out of interest: why have these values changed? 🤔

Reply:
here are the orders used https://github.com/shopware5/SwagPaymentPayPalUnified/blob/main/Tests/Functional/Components/TransactionReport/_fixtures/orders.sql

the paymentId `19` is now no longer assigned to a PayPal payment method, as there is now one less. Let me check if I can remove them

Reply:
I changed the test data, should work again, without changing the test :blush: